ui/js/component/app/index.js

```diff
@@ -2,10 +2,15 @@ import React from "react";
 import { connect } from "react-redux";
 import { selectPageTitle } from "selectors/navigation";
 import { selectUser } from "selectors/user";
-import { doCheckUpgradeAvailable, doAlertError } from "actions/app";
+import {
+  doCheckUpgradeAvailable,
+  doStartUpgradeCheckTimer,
```

Comment:
I dont think we ever use doStartUpgradeCheckTimer and also given some of my comments below we also might not need doCheckUpgradeAvailable

Reply:
Yep, my bad. I missed this one probably because the imports were folded in my IDE. I should've been more precise.